contracts/src/test/7683/t1BasicSwapE2E.t.sol

```diff
@@ -28,7 +28,8 @@ import {
     FillInstruction
 } from "intents-framework/ERC7683/IERC7683.sol";
```

Comment:
Would this file fit better in `scripts/test/` as `t1ERC7683PullE2E.s.sol`? The other way around could work as well.

Reply:
This doesn't make sense to me. I don't think that we should be writing scripts that require the relayer to relay messages in order for the script to work.